### PR TITLE
Replace mentions of "Deploy" with "Enable" to reflect UI changes

### PR DESCRIPTION
**Pull request by seialkali**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-86815
https://jira.d2iq.com/browse/D2IQ-87258

## Description of changes being made

The wording has now changed in the UI so that you "Enable" an application, rather than "Deploy". We need to update the docs to reflect this change. 

I've also updated some of the instructions to make them a little bit clearer.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4274.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.2/logging/enable-logging/enable-logging-via-ui/index.md [hunk 1]
```diff
@@ -9,28 +9,29 @@ beta: false
...
+To enable workspace-level logging in DKP using the UI, follow these steps:
 
-1. In the **Global workspace** nav, click on the **Global** dropdown and then select the Workspace you want to deploy for your logging stack.
+1.  From the top menu bar, select your target workspace.
 
-1. In the left rail nav, click on **Applications**.
+1.  Select **Applications** from the sidebar menu.
 
-1. Ensure that cert-manager and Traefik are enabled in the Workspace. Scroll down in the **Applications** page to view the **Foundational** applications section. Confirm that cert-manager and Traefik are deployed.
+1.  Traefik and cert-manager are required to be deployed for the logging stack. Ensure that Traefik is enabled in the Workspace, and cert-manager is enabled or already deployed to the cluster.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/logging/enable-logging/enable-logging-via-ui/index.md:20]:
```suggestion
1. Traefik and cert-manager are required to be deployed for the logging stack. Ensure that Traefik is enabled in the Workspace, and cert-manager is enabled or already deployed to the cluster.
```

File: pages/dkp/kommander/2.2/projects/project-network-policies/index.md [hunk 2]
```diff
@@ -39,10 +39,10 @@ This section also contains the **Pod Selector** fields for selecting pods using
 The **Policy Types** selections help to define the type of Network Policy you are creating:
 
 - **Default** - automatically includes ingress, and egress is set only if the network policy defines egress rules.
-- **Ingress** - this policy applies to ingress traffic for the selected pods and/or namespaces using the options you define below.
-- **Egress** - this policy applies to egress traffic for the selected pods and/or namespaces using the options you define below.
+- **Ingress** - this policy applies to ingress traffic for the selected pods, namespaces using the options you define below, or both.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/projects/project-network-policies/index.md:42]:
```suggestion
- **Ingress** - this policy applies to ingress traffic for the selected pods, to namespaces using the options you define below, or both.
```

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/application-deployment/index.md [hunk 3]
```diff
@@ -22,39 +22,40 @@ Set the `WORKSPACE_NAMESPACE` environment variable to the name of the workspace'
 export WORKSPACE_NAMESPACE=<workspace_namespace>
 ```
 
-After creating a GitRepository, use either the DKP UI or the CLI to deploy your catalog applications.
+After creating a GitRepository, use either the DKP UI or the CLI to enable your catalog applications.
+<p class="message--important"><strong>IMPORTANT: </strong>From within a workspace, you can enable applications to deploy, but to confirm that an application has successfully deployed you must <a href="#verify-applications">verify via the CLI</a>.</p>
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/application-deployment/index.md:26]:
```suggestion
<p class="message--important"><strong>IMPORTANT: </strong>From within a workspace, you can enable applications to deploy, but to confirm that an application has successfully deployed, you must <a href="#verify-applications">verify via the CLI</a>.</p>
```

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/application-deployment/index.md [hunk 4]
```diff
@@ -22,39 +22,40 @@ Set the `WORKSPACE_NAMESPACE` environment variable to the name of the workspace'
...
+1.  From the top menu bar, select your target workspace.
 
-1.  Select **Applications** on the left navigation bar to browse the available applications from your configured repositories.
+1.  Select **Applications** from the sidebar menu to browse the available applications from your configured repositories.
 
-1.  Select your desired application.
+1.  Select the three dot button from the bottom-right corner of the desired application tile, and then select **Enable**.
 
-1.  Select the version you'd like to deploy from the version drop-down, and then select Deploy. The `Deploy Workspace Catalog Application` page is displayed.
+1.  If available, select a version from the drop-down. This drop-down will only be visible if there is more than one version.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/application-deployment/index.md:38]: Checking with colleagues if my suggestion should be the standard. 
```suggestion
1.  If available, select a version from the drop-down menu. This drop-down menu will only be visible if there is more than one version.
```

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/deploy-custom-app/index.md [hunk 5]
```diff
@@ -1,58 +1,55 @@
...
+1.  From the top menu bar, select your target workspace.
 
-1. Select **Workspace** > **Workspace**
+1.  Select **Applications** from the sidebar menu to browse the available applications from your configured repositories.
 
-1. Select your workspace from the list.
+1.  Select the three dot button from the bottom-right corner of the desired application tile, and then select **Enable**.
 
-1. Select **Applications** on the top navigation bar to browse the available applications from your configured repositories.
+1.  If available, select a version from the drop-down. This drop-down will only be visible if there is more than one version.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/deploy-custom-app/index.md:22]:
```suggestion
1.  If available, select a version from the drop-down menu. This drop-down menu will only be visible if there is more than one version.
```

File: pages/dkp/kommander/2.2/workspaces/applications/platform-applications/application-deployment/index.md [hunk 6]
```diff
@@ -27,13 +27,15 @@ Set the `WORKSPACE_NAMESPACE` environment variable to the name of the workspace'
 export WORKSPACE_NAMESPACE=<workspace_namespace>
 ```
 
-## Deploy the application
+<p class="message--important"><strong>IMPORTANT: </strong>From the CLI, you can enable applications to deploy, but to confirm that an application has successfully deployed you must <a href="#verify-applications">verify via the CLI</a>.</p>
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/workspaces/applications/platform-applications/application-deployment/index.md:30]:
```suggestion
<p class="message--important"><strong>IMPORTANT: </strong>From the CLI, you can enable applications to deploy. Verify that an application has successfully deployed <a href="#verify-applications">via the CLI</a>.</p>
```

File: pages/dkp/kommander/2.2/workspaces/applications/platform-applications/index.md [hunk 7]
```diff
@@ -56,3 +58,5 @@ The following table describes the list of platform applications that are deploye
 <p class="message--note"><strong>NOTE: </strong>Only a single deployment of <code>traefik</code> per cluster is supported.</p>
 
 <p class="message--note"><strong>NOTE: </strong>Kommander automatically manages the deployment of <code>traefik-forward-auth</code> and <code>kube-oidc-proxy</code> when clusters are attached to the workspace. These applications are not shown in the DKP UI.</p>
+
+<p class="message--note"><strong>NOTE: </strong>Applications are enabled in DKP and then deployed to attached clusters. To confirm that your enabled application has successfully deployed, you should <a href="../platform-applications/application-deployment#verify-applications">verify via the CLI</a>.</p>
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/workspaces/applications/platform-applications/index.md:62]:
```suggestion
<p class="message--note"><strong>NOTE: </strong>Applications are enabled in DKP and then deployed to attached clusters. Verify that an application has successfully deployed <a href="#verify-applications">via the CLI</a>.</p>

File: pages/dkp/kommander/2.2/projects/applications/catalog-applications/custom-applications/deploy-custom-app/index.md [hunk 8]
```diff
@@ -23,37 +22,37 @@ Go to the DKP UI to deploy your custom applications:
 
 1.  Select the three dot button from the bottom-right corner of the desired application tile, and then select **Enable**.
 
-1.  Select the version you'd like to enable from the version drop-down, and then select **Enable**. This drop-down will only be visible if there is more than one version available.
+1.  If available, select a version from the drop-down. This drop-down will only be visible if there is more than one version.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.2/projects/applications/catalog-applications/custom-applications/deploy-custom-app/index.md:25]:
```suggestion
1.  If available, select a version from the drop-down menu. This drop-down menu will only be visible if there is more than one version.
```

**Review by nataliepina — APPROVED** [review]

**Pull request merged**